### DISCUSSION TITLE
pnpm as alternative to yarn

### DISCUSSION THREAD

**Discussion by indicozy** [chen-rn/CUA#55] [Ideas]
Hi there!

I'm still digging through your code, and I found out that you are using yarn. I believe that pnpm is better suited for monorepo due to its work with workspaces. 

Some reference: https://nhost.io/blog/how-we-configured-pnpm-and-turborepo-for-our-monorepo

What do you think?

**Comment by albbus-stack** [comment]
Somewhere else @chen-rn was already discussing this. I am strongly in favour of switching to pnpm since it's the fastest package manager out there right now. It should be really easy to migrate to pnpm, if you want to make a new branch, you are more than welcome 😍

**Comment by indicozy** [comment]
Gotcha, wait for pr
